treeherder/log_parser/tasks.py

```diff
@@ -143,3 +150,53 @@ def extract_text_log_artifacts(job_log):
         )
 
     return artifact_list
+
+
+def post_perfherder_artifacts(job_log):
```

Comment:
@junngo I think it would be better for us to put this into a separate area. This folder seems to be specifically for parsing logs, but we're parsing JSONs instead. What do you think about having this task defined here in the perf directory? https://github.com/mozilla/treeherder/blob/505ad6b4047f77fc3ecdea63e57881116340d0fb/treeherder/perf/tasks.py

Reply:
@gmierz Splitting the code is a great idea. Creating a separate file under the code directory [0] looks good to me. It feels more cohesive to put it there, since the log parsing [1] also lives in that folder.
Please consider my opinion and feel free to tell me about the directory location.

[0]
https://github.com/mozilla/treeherder/tree/505ad6b4047f77fc3ecdea63e57881116340d0fb/treeherder/log_parser
[1]
https://github.com/mozilla/treeherder/blob/505ad6b4047f77fc3ecdea63e57881116340d0fb/treeherder/log_parser/artifactbuildercollection.py#L85

Reply:
I added the new file based on your feedback. It seems more suitable since the JSON artifact isn’t part of the log parsing process :)